src/lib/component/Icon.svelte

```diff
@@ -13,5 +13,5 @@
     <Icon {src} {color} {size} />
   </a>
 {:else}
-  <Icon {src} {color} {size} />
+  <Icon {src} {size} color="currentcolor" className={$$props.color} />
```

Comment:
Why do you use `$$props.color`? Isn't simply `color` the same? Maybe I'm missing something here

Reply:
I'd also appreciate an explanation about className, it just reminds me of react xd

Reply:
> I'd also appreciate an explanation about className, it just reminds me of react xd

![image](https://github.com/NIAEFEUP/website-niaefeup-frontend/assets/93883163/0f5ddcb7-6d51-4226-b786-d334404350c5)

It's just how the icon pack declares it ahah, otherwise I get a `class does not exist in type` error

Reply:
> Why do you use `$$props.color`? Isn't simply `color` the same? Maybe I'm missing something here

Yup! Will change it. Also thinking of instead of this line simply adding a `className` property, for when the color trying to be used is not recognized by the `color` property (eg. custom names, different opacity notation). This way the color wouldn't have to be something like `fill-rose-950` and could actually just be a hex or color xd. What do you think?

Reply:
@MRita443 I think `className` is specific to the `Icon` library we use. I prefer to use `$$props.class` and map it to `className` in `Icon.svelte`, so we have the same syntax for every component. But I'm willing to hear more opinions